vcr/api/v2/api.go

```diff
@@ -118,7 +140,7 @@ func (w *Wrapper) VerifyVC(ctx echo.Context) error {
 		return err
 	}
 
-	if err := w.VCR.Validate(requestedVC, true, true, nil); err != nil {
+	if err := w.VCR.Validate(requestedVC, false, true, nil); err != nil {
```

Comment:
does this then still work for `NutsAuthorizationCredential`s, which are never trusted?

Reply:
No, it will not work. Added extra options for this API request.